content/docs/command-reference/exp/init.md

```diff
@@ -191,8 +182,6 @@ train:
   deps:
     - data/features
     - src/train.py
-  params:
-    - epochs
```

Comment:
We should be keeping `params:` but dropping `epochs`. So it should look like:

```yaml
train:
  cmd: python src/train.py
  deps:
    - data/features
    - src/train.py
  params:
  outs:
    - models/predict.h5
  metrics:
    - metrics.json:
        cache: false
```

Reply:
That's the default now? OK, done.